kart/conflicts.py

```diff
@@ -216,9 +216,20 @@ def item_to_text(key, value, path, level):
 def conflicts_json_as_geojson(json_obj):
     """Converts the JSON output of list_conflicts to geojson."""
     features = []
+    meta_item_key = ""
     for key, feature in json_obj.items():
+        if type(feature) != dict or feature.get("type") != "Feature":
+            meta_item_key = key
+            continue
         feature["id"] = key
         features.append(feature)
+
+    if meta_item_key:
```

Comment:
This prints out the last encountered meta-item conflict, regardless of how many there are.
I recommend instead one of the following options: 
- print out a single warning (in this case, don't bother to explain what they are at all)
- print out a warning for each dataset with meta-item conflicts (in this case, give the path of the dataset that has a meta-item conflict)
- print out a warning for each meta-item conflict (in this case, print the entire key of the meta-item, but not the version part of it eg ancestor, ours, theirs)

Reply:
> print out a warning for each meta-item conflict (in this case, print the entire key of the meta-item, but not the version part of it eg ancestor, ours, theirs)

I have made some changes according to this. Let me know if it's what you meant.